.circleci/config.yml

```diff
@@ -11,34 +11,46 @@ workflows:
       - codacy/checkout_and_version:
           write_sbt_version: true
       - codacy/sbt:
-          name: publish_docker_local
+          name: test_and_coverage
+          persist_to_workspace: true
           cmd: |
-            sbt "set scalafmtUseIvy in ThisBuild := false;
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v8.11.3
+            nvm alias default v8.11.3
+
+            npm install
+            export PATH=$PATH:$(pwd)/node_modules/.bin
+            STYLELINT_CONFIG_BASEDIR=$(pwd)/node_modules sbt "set scalafmtUseIvy in ThisBuild := false;
                  scalafmt::test;
                  test:scalafmt::test;
                  sbt:scalafmt::test;
-                 set name := \"$CIRCLE_PROJECT_REPONAME\";
+                 coverage;
+                 test;
+                 coverageReport;
+                 coverageAggregate"
```

Comment:
You can remove `coverageAggregate` since there is only one sub-project.
In any case coverageAggregate should go [in another sbt run](https://github.com/scoverage/sbt-scoverage/blob/60d4ef9af9d078d6413f6ab5d109e3d094d17420/README.md#multi-project-reports).

Reply:
Removed it ✅